main.go

```diff
@@ -13,13 +13,14 @@ func main() {
 		fmt.Fprintf(os.Stderr, "URL parameter not informed\n")
 		os.Exit(1)
 	}
-	url := os.Args[1]
-	resp, err := furl.Get(url)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Error fetching URL:%v\n", err)
-		os.Exit(1)
-	}
-
-	fmt.Printf("%dms %7d %s %s\n", resp.ElapsedTime, resp.NBytes, resp.Url, resp.Body)
+	for i := 1; i < len(os.Args); i++ {
```

Comment:
Você acha mesmo uma boa ideia fazer um for no main do projeto?
Acho que poderia trabalhar a função GET do package furl.